### PR TITLE
#10 - clean up document targets and components

### PR DESCRIPTION
**Pull request by oieduardorabelo**
addressing discussion from https://github.com/testing-library/svelte-testing-library/issues/10

### CODE REVIEW

File: tests/render.spec.js [hunk 1]
```diff
@@ -61,4 +65,8 @@ describe('render', () => {
     expect(getByText('Hello world!')).toBeInTheDocument()
     expect(getByTestId('custom-target')).toBeInTheDocument()
   })
+
+  test('after each test above, document is clean from targets and components', () => {
+    expect(document.body.innerHTML).toBe('<div></div>')
```
Comment (oieduardorabelo) [inline tests/render.spec.js]: output here is dependent on previous tests,

- before: `<div></div><div></div><div></div><div></div><main data-testid=\"custom-target\"></main>`

File: tests/render.spec.js [hunk 2]
```diff
@@ -38,13 +38,17 @@ describe('render', () => {
...
+
     global.console = {log: jest.fn()}
 
     const {debug} = render(App)
 
     debug()
 
     expect(global.console.log).toHaveBeenCalledWith(prettyDOM(document.body))
+
+    global.console = originalConsole
```
Comment (oieduardorabelo) [inline tests/render.spec.js]: i wasn't able to use `console.log` after this test case, just preserving the env global here

File: tests/render.spec.js (hunk 1 above)
Comment (EmilTholin) [reply tests/render.spec.js]: I think the body will be entirely empty after this great addition.

File: tests/render.spec.js (hunk 2 above)
Comment (benmonro) [reply tests/render.spec.js]: Thank you!  I was just thinking about that over breakfast lol

File: tests/render.spec.js (hunk 1 above)
Comment (benmonro) [reply tests/render.spec.js]: I agree with @emiltholin. It should be empty

Comment (oieduardorabelo) [reply tests/render.spec.js]: indeed is happening, as we can see in the failed build on travis:

- https://travis-ci.org/testing-library/svelte-testing-library/builds/528192604#L503

i bet it was some kind of local jest cache, running with `--no-cache` seems to fixe it!

thanks @EmilTholin @benmonro

**Pull request merged**